quickjs.c

```diff
@@ -42193,14 +42193,29 @@ static JSValue js_math_random(JSContext *ctx, JSValueConst this_val,
     return __JS_NewFloat64(ctx, u.d - 1.0);
 }
 
+static double qjs_ceil(double x)
```

Comment:
Looking closer at the QuickJS style, it looks like we should fix two inconsistencies here:
- namespace these like the other functions here, e.g. `js_math_ceil()` instead of `qjs_ceil()`
- make sure the function definitions are in the same order here as they are down in `js_math_funcs`

Reply:
uhm not really. the js_* functions are javascript functions. what im wrapping here is libc functions

Reply:
oh well. seems like js_math is also used to wrap c functions. so its a bit inconsistent. but ill follow their inconsistency

Reply:
Actually the convnetion is just prefix them with `js_` because the direct calls are accessed thru the `f_f` and `f_f_f` functions and those are the ones that handle the js context instead